1/homework/HOWTO.txt

```diff
@@ -0,0 +1,4 @@
+python [data_path] [target_path] [noise_path] [music_alpha] [beep_alpha] [beep_frequency]
```

Comment:
Please add requirements.txt so I would know which version on packages to use
This is especially important for librosa